packages/blocks/src/api/raw-handling/utils.js

```diff
@@ -37,18 +37,41 @@ export function getBlockContentSchema( transforms ) {
 				schema[ tag ].attributes.push( 'id' );
 			}
 		}
+
+		// If an isMatch function exists add it to each schema tag that it applies to.
+		if ( isMatch ) {
+			for ( const tag in schema ) {
+				schema[ tag ].isMatch = isMatch;
```

Comment:
Is this mutating the block type? Should it?

Reply:
Hi @mcsf, the transforms passed to this function are always the result of getRawTransformations and this function copies the transforms, so we are mutating that copy and not the block type.
https://github.com/WordPress/gutenberg/blob/fix/media-text-i18n/packages/blocks/src/api/raw-handling/index.js#L57-L65
This function was already mutating the input, but I think given that it is a getter it is unexpected that this function mutates the input and I will open a follow-up PR that removes the mutations.